packages/datetime/src/dateRangePicker.tsx

```diff
@@ -237,9 +237,12 @@ export class DateRangePicker extends AbstractPureComponent2<DateRangePickerProps
         return (
             <div className={classes}>
                 {this.maybeRenderShortcuts()}
-                <div>
-                    {this.renderCalendars(isShowingOneMonth)}
-                    {this.maybeRenderTimePickers()}
+                <div className={DateClasses.DATEPICKER_CONTENT}>
```

Comment:
also this CSS class was named for and designed for "datepicker", not "date range picker", so we'll need to make a new class to keep the naming semantics clear